paasta_tools/cli/cmds/rollback.py

```diff
@@ -41,8 +76,8 @@ def add_subparser(subparsers):
     list_parser.add_argument(
         '-k', '--commit',
         help='Git SHA to mark for rollback',
```

Comment:
Can you expand the help doc to explain the default behavior here?
Also can you be explicit about the default?